code/game/machinery/computer/tram_controls.dm

```diff
@@ -7,7 +7,6 @@
 	flags_1 = NODECONSTRUCT_1
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
 
-	var/obj/structure/industrial_lift/tram/tram_part
```

Comment:
This seems like an unnecessary change that makes it harder to iterate on in the future. Are you sure you need this?

Reply:
I did this because it was already kinda tied to how things work currently. I'm up for converting it to weakrefs, will do that tomorrow

Reply:
done